index.css

```diff
@@ -385,6 +350,153 @@ li {
   margin-top: 1em;
 }
 
+.modal {
+  display: none;
+  position: fixed;
+  border-radius: 20px;
+  z-index: 10;
+  top: 0;
+  width: 90%;
+  overflow: auto;
+  background-color: rgb(0, 0, 0);
+  background-color: rgba(0, 0, 0, 0.4);
+  left: 0;
+  margin: 0;
+}
+
+.modal-buttons {
+  display: flex;
+  justify-content: space-between;
+}
+
+.close-modal {
+  display: inline !important;
+  color: var(--title-color);
+  font-size: x-large;
+  margin: 0;
+}
+
+.modal-title-close h2 {
+  width: fit-content;
+  margin: 0;
+  display: inline-block;
+}
+
+.modal-title-close {
+  display: flex;
+  align-items: baseline;
+  justify-content: space-between;
+  width: 100%;
+  padding: 0;
+  margin: 0;
+  height: 50px;
+}
+
+.modal-top {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  gap: 1em;
+}
+
+@media (min-width: 375px) {
+  .modal {
+    display: none;
+    position: fixed;
+    border-radius: 20px;
+    z-index: 10;
+    top: 0;
+    width: 100vw;
+    height: 100vh;
+    overflow: auto;
+    background-color: rgb(0, 0, 0);
+    background-color: rgba(0, 0, 0, 0.4);
+    left: 0;
+    margin: 0;
+    padding: 0;
+    animation: popupSlideDown 0.3s;
+  }
```

Comment:
- [ ] Same thing applies to this. Kindly look into it and fix.

### Your Modal Mobile
![your-mobile-card](https://user-images.githubusercontent.com/26597432/223993168-7a1b9a6c-3113-47f8-a6d4-b799159fdef8.png)

### Figma Modal Mobile
![temp-mobile-card](https://user-images.githubusercontent.com/26597432/223993246-d78b694a-6c59-4ec3-ab31-7d2be1a14340.png)